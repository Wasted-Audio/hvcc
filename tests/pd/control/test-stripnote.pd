#N canvas 722 308 249 251 12;
#X obj 58 25 loadbang;
#X obj 58 50 t b b;
#X msg 58 85 23 0;
#X msg 101 85 34.5 67.8;
#X obj 58 120 stripnote;
#X obj 58 152 print;
#X connect 0 0 1 0;
#X connect 1 0 2 0;
#X connect 1 1 3 0;
#X connect 2 0 4 0;
#X connect 3 0 4 0;
#X connect 4 0 5 0;
#X connect 4 1 5 0;
