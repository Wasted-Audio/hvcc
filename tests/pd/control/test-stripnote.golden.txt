[@ 0.000] print: 20
[@ 0.000] print: 10
[@ 0.000] print: 50
[@ 0.000] print: 40
[@ 0.000] print: 50
[@ 0.000] print: 15
[@ 0.000] print: 67.8
[@ 0.000] print: 34.5
